repose-aggregator/commons/commons-utilities/src/main/java/org/openrepose/commons/utils/logging/apache/format/converters/DateConversionFormat.java

```diff
@@ -19,27 +19,21 @@
  */
 package org.openrepose.commons.utils.logging.apache.format.converters;
 
-public enum DateConversionFormat {
+public class DateConversionFormat {
```

Comment:
Not sure how i feel about this one. It doesn't quite feel right in it's new form.